Source/WebCore/platform/graphics/cg/PathCG.cpp

```diff
@@ -310,6 +321,20 @@ static inline void addToCGContextPath(CGContextRef context, PathRoundedRect segm
     CGContextSetCTM(context, ctm);
 }
 
+static inline void addToCGContextPath(CGContextRef context, PathContinuousRoundedRect segment)
+{
+    // No API to add continuous rounded rects to context.
+    auto path = PathCG::create();
```

Comment:
Prefer explicit smart pointer:
```suggestion
    Ref path = PathCG::create();
```